sdk/core/supported-version-matrix.json

```diff
@@ -13,10 +13,10 @@
     "AZURE_TEST_HTTP_CLIENTS": "netty",
     "TestGoals": "surefire:test",
     "AZURE_CORE_TEST_SUPPORTED_JACKSON_VERSION": [
-        "2.10.5",
-        "2.11.4",
-        "2.12.5",
-        "2.13.2"
+        "2.10.0",
+        "2.11.0",
+        "2.12.0",
+        "2.13.0"
```

Comment:
Why did we downgrade the versions here?

Reply:
My thoughts are we should be validating supported versions against the lowest patch version in the minor version to ensure maximum compatibility